examples/hybrid_job.py

```diff
@@ -0,0 +1,50 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+from braket.aws import AwsDevice
+from braket.circuits import Circuit, FreeParameter, Observable
+from braket.devices import Devices
+from braket.jobs import hybrid_job
+from braket.jobs.metrics import log_metric
+
+device_arn = Devices.Amazon.SV1
+
+
+@hybrid_job(device=device_arn)  # choose priority device
+def run_hybrid_job(num_tasks=1):
+    device = AwsDevice(device_arn)  # declare AwsDevice within the hybrid job
+
+    # create a parametric circuit
+    circ = Circuit()
+    circ.rx(0, FreeParameter("theta"))
+    circ.cnot(0, 1)
+    circ.expectation(observable=Observable.X(), target=0)
+
+    theta = 0.0  # initial parameter
+
+    for i in range(num_tasks):
+        task = device.run(circ, shots=100, inputs={"theta": theta})  # input parameters
+        exp_val = task.result().values[0]
+
+        theta += exp_val  # modify the parameter (possibly gradient descent)
+
+        log_metric(metric_name="exp_val", value=exp_val, iteration_number=i)
+
+    return {"final_theta": theta, "final_exp_val": exp_val}
+
+
+job = run_hybrid_job(num_tasks=5)
+print(job)
```

Comment:
why print this? I suggest we just use wait_until_complete=True

Reply:
its just print the job ARN, not the whole logging

Reply:
I believe we want to print the logging, since this is an introductory example